src/igvfd/schemas/auxiliary_set.json

```diff
@@ -1,6 +1,6 @@
 {
     "title": "Auxiliary Set",
-    "description": "A file set for auxiliary raw data files that were produced alongside raw data files from a measurement set. For example, in a CRISPR screen experiment the measurement set would capture the result of single-cell transcript sequencing and the auxiliary set the result of gRNA sequencing with the associated cellular barcodes.",
+    "description": "Auxiliary Set is a file-set that hosts raw data files (e.g. FASTQs) that result from sequencing of nucleic acids from a sample that are a proxy to some vital information and necessary for the analysis of the experiment it is connected to in the Measurement Set. Auxiliary sets usually would not provide any information about the transcriptome or the genome of the sample in question. For example AuxiliarySets would include the sequencing of barcodes that correspond to elements introduced into cells, or sequencing of guide RNA coding sequences in the cells. The files hosted in the AuxiliarySets are relevant for the Analysis, but they by themselves are not assessig much of the biology of the sample being analyzed.",
```

Comment:
* I still prefer not capitalizing the schema names. I think it should say things like "measurement set" or "auxiliary set" not "AuxiliarySet".
* Also, I think we previously have not used a hyphen in "file set"
* "the analysis of the experiment it is connected to in the Measurement Set" is a little wordy and confusing to me. Does it just mean "the analysis of the associated measurement set"?
* Typo "assessig" should be "assessing"